app/src/main/java/org/p2p/wallet/kyc/model/StrigaKycSignUpStatus.kt

```diff
@@ -0,0 +1,65 @@
+package org.p2p.wallet.kyc.model
+
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import org.p2p.wallet.R
+
+enum class StrigaKycSignUpStatus(
```

Comment:
It's not clear that the model used for banners, consider to rename it to something like:
- StrigaKycBannerStatus
- StrigaKycSignupBannerStatus
- StrigaKycSignupBannerData
- other options with *banner*

Btw, we use `Signup` instead of `SignUp`